keyboards/tada68/keymaps/mekberg/keymap.c

```diff
@@ -0,0 +1,99 @@
+#include QMK_KEYBOARD_H
+
+// Fillers to make layering more clear
+#define XXXXXX KC_TRNS
+#define ______ KC_NO
```

Comment:
QMK defines seven-character keycodes for these automatically.

https://github.com/qmk/qmk_firmware/blob/a120a000d481ed14a87988383be46c5a76604bb6/quantum/quantum_keycodes.h#L29-L31

If you use the native ones in your keymap, you don't need these lines.

Reply:
If you notice, I actually swap them because I wanted KC_NO on my layers, but all the XXXXXX make it really unreadable. I can add a comment to make this more clear.

Reply:
Good call; I actually didn't notice the swap of `______` being `KC_NO`.